gate-web/gate-web.gradle

```diff
@@ -21,6 +21,7 @@ repositories {
 dependencies {
   compile project(":gate-core")
   compile project(":gate-proxy")
+  compile project(":gate-integrations-gremlin")
```

Comment:
I didn't see any way around this.  I need the `@Configuration` of `GremlinGateConfig` to get detected, so I need those classes visible to the webapp.

Reply:
👍 Yes, there's no way around this right now. 

Subsequent integrations we could probably wire something up that would automatically add dependencies on anything matching a `*integrations*` pattern.